nixos/modules/services/misc/gitea.nix

```diff
@@ -289,9 +322,6 @@ in
         runConfig = "${cfg.stateDir}/custom/conf/app.ini";
         secretKey = "${cfg.stateDir}/custom/conf/secret_key";
       in ''
-        # Make sure that the stateDir exists, as well as the conf dir in there
-        mkdir -p ${cfg.stateDir}/conf
-
         # copy custom configuration and generate a random secret key if needed
         ${optionalString (cfg.useWizard == false) ''
           mkdir -p ${cfg.stateDir}/custom/conf
```

Comment:
we can move that line to tmpfiles, too?

Reply:
It looks like this directory is created regardless of `cfg.useWizard` so I unconditionally added it to keep the configuration simple.